src/Foundation/NSProgress.cs

```diff
@@ -0,0 +1,30 @@
+﻿#if !COREBUILD && XAMCORE_2_0
+using System;
+using XamCore.Foundation;
+using XamCore.ObjCRuntime;
+
+namespace XamCore.Foundation {
+	public partial class NSProgress {
+		//Manual bindings until BindAs support is merged
+		public nint? EstimatedTimeRemaining {
+			get { return _EstimatedTimeRemaining == null ? null : (nint?)_EstimatedTimeRemaining.NIntValue; }
```

Comment:
This fetches `_EstimatedTimeRemaining` twice (which is an expensive managed->native transition), I think it would be better to cache the `_EstimatedTimeRemaining` value locally.

Or does this work: `return _EstimatedTimeRemaining?.NIntValue`?